src/packages/engine-core/src/NAPIEngine.ts

```diff
@@ -458,10 +468,7 @@ You may have to run ${chalk.greenBright(
   ): Promise<{ data: T; elapsed: number }> {
     try {
       await this.start()
-      debug(`request: ${this.connected}`)
-      if (!this.connected) {
-        await this.start()
-      }
+      debug(`state:${this.connected}`)
```

Comment:
```suggestion
      debug(`request state: ${this.connected}`)
```

Since it happens in `request`